tests/integration/test_playlist_routes.py

```
@@ -93,3 +93,74 @@ def test_toggle_only_fresh_and_snooze(client, device_config_dev):
+        "end_time": "10:00",
+    }
+    resp = client.put("/update_playlist/Morning", json=upd)
+    assert resp.status_code == 200
+    data = resp.get_json()
+    assert data.get("success") is True
+    assert "warning" in data
+    assert "Default" in data["warning"]
+    assert "priority" in data["warning"]
+
+
+def test_update_default_playlist_does_not_warn_about_itself(client, device_config_dev):
```

Ruff (0.15.9)

[warning] Unused function argument: `device_config_dev`

(ARG001)